routes/truckRoutes.js

```diff
@@ -1 +1,105 @@
-// put your Express router code in here
\ No newline at end of file
+'use strict';
+// put your Express router code in here
+
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-undef */
+
+var express = require('express');
+var foodTrucks = require('../trucks');
+
+var router = express.Router();
+
+var trucks = require('../trucks');
+var foodTypes = require('../trucks');
+var openTrucks = require('../trucks');
+var typeTrucks = require('../trucks');
+/* eslint-enable no-undef */
+
+// var serveStatic = express.static('public');
+
+var axeTruck = 'Fez';
+
+var openTruck = trucks.filterTrucksByDay('Saturday');
```

Comment:
Are these lines just some testing you were doing? I'm not sure what this code is for.